packages/venia-concept/src/util/makeMediaUrl.js

```diff
@@ -0,0 +1,37 @@
+import memoize from 'lodash/memoize';
+
+const absoluteUrl = /^https?:\/\//i;
+
+const joinUrls = (base, url) =>
+    (base.endsWith('/') ? base.slice(0, -1) : base) +
+    '/' +
+    (url.startsWith('/') ? url.slice(1) : url);
+
+const mediaPathPrefixes = {
+    product: '/media/catalog/product/',
```

Comment:
These should be provided from env by a Webpack EnvironmentPlugin, so that we can swap out URL generation schemes from a central location.